plugins/chat/db/migrate/20230411032630_add_thread_not_deleted_index_chat_messages.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddThreadNotDeletedIndexChatMessages < ActiveRecord::Migration[7.0]
```

Comment:
Small note, it is more beneficial to run this migration first since it'll benefit the subsequent migration being added here.

Reply:
Sure, can move it earlier